WordPress/gutenberg
# [RNMobile] Bump AztecAndroid version

## What?
Bumping the version of the AztecAndroid dependency

## Why?
This brings in a significant performance improvement.

The PR description of [the related WordPress Android PR](https://github.com/wordpress-mobile/WordPress-Android/pull/16490) contains more information.

## Testing Instructions
1. Verify that the demo app runs without any issues
2. Follow the testing instructions in [the associated WordPress Android PR](https://github.com/wordpress-mobile/WordPress-Android/pull/16490).
